R/await.R

```diff
@@ -178,18 +179,19 @@ safe_call_once <- function(...) {
   tryCatch(call_once(...), error = function(e) e)
 }
 
-# .id is the first argument to f.
-call_once <- function(f, ..., .id = NULL, .status_key = "state",
+# .id is the first argument to f
+call_once <- function(f, ...,
+                      .status_key = "state",
                       .success_states = c("succeeded"),
                       .error_states = c("failed", "cancelled"), fname) {
-  response <- do.call(f, c(.id, list(...)))
+  response <- do.call(what = f, args = sapply(X = ..., FUN = list))
```

Comment:
Removing `.id` is right. I think we should be able to do something like

```
f <- function(x, y) c(x, y)
do.call(f, list(1, 2))
```

Note that it won't work if the args are named
```
do.call(f, list(a=1, b=2))
Error in (function (x, y)  : unused arguments (a = 1, b = 2)
```